PicIt/Model/Countdown.swift

```diff
@@ -0,0 +1,238 @@
+//
```

Comment:
`Countdown` replaces what was `PicItTimer`. It's a much cleaner approach (and fixed some annoying bugs along the way).